taskcluster/ffios_taskgraph/routes.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Comparing against fenix counterpart, it LGTM, including the spefici de-duplication.
 
Offtopic: this makes me wonder how come I don't yet have this module in application-services and whether I should have.